addon/utils/lookup-classes.js

```diff
@@ -0,0 +1,5 @@
+import data from '../css-classes-json';
```

Comment:
We should avoid using relative path given the output directory specificity https://github.com/Ticketfly-UI/ember-css-composer/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR46